### PR TITLE
fix: add documentation and transform object to #24 guide

### PR DESCRIPTION
**Pull request by kallyas**
This PR adds documentation and properties to the following objects based on #24 guide

- [x] File
- [x] InputMedia
- [x] Update
- [x] User
- [x] UserProfilePhotos    
- [x] LoginUrl
- [x] Location
- [x]  ProximityAlertTriggered
- [x] ReplyKeyboardMarkup
- [x] ReplyKeyboardRemove
- [x] ResponseParameters
- [x] Venue
- [x] Video
- [x] VideoChatEnded
- [x] VideoChatParticipantsInvited
- [x] VideoChatScheduled
- [x] VideoNote
- [x] Voice

### CODE REVIEW

**Comment by ht-thomas** [comment]
Thanks for this PR. However I'm noticing as if you're excluding some properties from the classes and only leaving there a few.

File: telegram/objects/file.py [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
+        
+        file_size (int): Optional. File size, if known
+        
+        file_path (str): Optional. File path. Use https://api.telegram.org/file/bot<token>/<file_path> to get the file.
+    """
+
+    __slots__ = (
+        'file_size', 
+        'file_path'
+        )
```
Comment (ht-thomas) [inline telegram/objects/file.py:20]: `file_size` and `file_path` need to be listed here as well.

**Review by ht-thomas — CHANGES_REQUESTED** [review]

File: telegram/objects/file.py (hunk 1 above)
Comment (kallyas) [reply telegram/objects/file.py:20]: I thought they are optional,

Comment (ht-thomas) [reply telegram/objects/file.py:20]: Yes, they are optional as telegram properties, but they still have to be present in python or else we shall have attribute errors when we try accessing them. So basically, every data member of the class must appear in the `__slots__` variable for it to be created in the `__init__` function.

Comment (kallyas) [reply telegram/objects/file.py:20]: Okay, so if I got you right, It should be something like this.

```python
class SomeClass(BaseObject):
         __slots__ = (
         'all properties here'
        )

      def __init__(self, all_properties_as_args):
               self.all_properties_as_args = all_properties_as_args
```

Comment (ht-thomas) [reply telegram/objects/file.py:20]: It's coming more this
``` python

class TestObject:

    __slots__ = ("a","b","c","d") #can be written in column form.

    def __init__ (self,x,y):
        self.a = x
        self.b = y
        self.c = None
        self.d = None
```
As you see, data members listed in slots are also declared in the init function with the same name. The init arguments are just regular arguments of any function.
I can't explain a lot here, but try reading online about slots to get more info on how they work.

**Review by ht-thomas — APPROVED** [review]

**Pull request merged**